app/app.go

```diff
@@ -582,7 +598,7 @@ func NewApp(
 		AddRoute(lpfarmtypes.RouterKey, lpfarm.NewFarmingPlanProposalHandler(app.LPFarmKeeper)).
 		AddRoute(exchangetypes.RouterKey, exchange.NewProposalHandler(app.ExchangeKeeper)).
 		AddRoute(ammtypes.RouterKey, amm.NewProposalHandler(app.AMMKeeper)).
-		AddRoute(liquidfarmingtypes.RouterKey, liquidfarming.NewProposalHandler(app.LiquidFarmingKeeper))
+		AddRoute(liquidammtypes.RouterKey, liquidamm.NewProposalHandler(app.LiquidAMMKeeper))
```

Comment:
Rather than replacing the router of the existing module liquidfarming, it would be better to add liquidamm separately and make an exception so that it does not work on liquidfarming's handler